### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by drowsy810301**
- urlize announcements link
- remove announcements background color
- footer 2014 to 2015
- upcoming contests order by start time
- remove input read-only attr
- end time cannot less than start time

### CODE REVIEW

File: index/forms.py [hunk 1]
```diff
@@ -33,3 +33,10 @@ class AnnouncementCreationForm(forms.ModelForm):
     class Meta:
         model = Announcement
         fields = ('content', 'start_time', 'end_time')
+
+    def clean_end_time(self):
+        start_time = self.cleaned_data.get("start_time")
+        end_time = self.cleaned_data.get("end_time")
+        if end_time <= start_time:
+            raise forms.ValidationError("End time cannot less than start time.")
```
Comment (nickelcheng) [inline index/forms.py]: "End time cannot be earlier than start time."

**Comment by drowsy810301** [comment]
comment fixed

**Comment by nickelcheng** [comment]
+1, @bruce3557 please check

**Comment by bruce3557** [comment]
+1

**Pull request merged**